tests/virt_autotest/install_package.pm

```diff
@@ -17,13 +17,15 @@ use testapi;
 
 sub install_package {
     my $qa_server_repo = get_var('QA_HEAD_REPO', '');
-    if ($qa_server_repo) {
-        script_run "zypper --non-interactive rr server-repo";
-        assert_script_run("zypper --non-interactive --no-gpg-check -n ar -f '$qa_server_repo' server-repo");
-    }
-    else {
-        die "There is no qa server repo defined variable QA_HEAD_REPO\n";
+    if ($qa_server_repo eq '') {
+        #default repo according to version if not set from testsuite
+        $qa_server_repo = 'http://dist.nue.suse.com/ibs/QA:/Head/SLE-' . get_var('VERSION');
```

Comment:
I generally do not like the idea of hard-coding URL generation inside a test. Why can't we just define QA_HEAD_REPO as a required variable for this test and add it to the medium/scenario?

Reply:
not really, will explain later :)

Reply:
Of course if it can be added to medium sle15 as sle12sp3, then this change will not be needed. Do you agree to add it to medium?

If not, then i need to specify on every testsuite , about 40, so terrible work :(. Since QA repo just stays as that for such a long time and do not see its change plan. So I  do it this way.

Reply:
Yes. We also discussed this offline: I think your approach is the best. I don't think it should be added to the medium although we have some repositories there which I don't like because the setting will confuse people when it's mentioned on each job but not needed. Also it's easier to review and follow when we go through PR review